site2/website-next/versioned_docs/version-2.7.3/administration-geo.md

```diff
@@ -91,6 +91,14 @@ $ bin/pulsar-admin clusters create \
 
 3. Run similar commands on `us-east` and `us-cent` to create connections among clusters.
 
+:::tip
+
+Different clusters may have different authentications.
+You can add auth param `--auth-plugin` and `--auth-parameters` to set cluster authentication, which will override `brokerClientAuthenticationPlugin` and `brokerClientAuthenticationParameters` if `authenticationEnabled=true`.\
+For more information about authentication, see [here](concepts-authentication.md).
+
+:::
+
```

Comment:
No need to update docs in `website-next` folder.

Reply:
Done

Reply:
Done